tools/packaging/packaging.psm1

```diff
@@ -22,22 +22,23 @@ function Start-PSPackage {
         [ValidateSet("deb", "osxpkg", "rpm", "msi", "zip", "AppImage", "nupkg")]
         [string[]]$Type,
 
-        # Generate windows downlevel package
-        [ValidateSet("win81-x64", "win7-x86", "win7-x64")]
+        [ValidateSet("win7-x64", "win7-x86")]
         [ValidateScript({$Environment.IsWindows})]
-        [string]$WindowsDownLevel,
+        [string]$WindowsRuntime,
 
         [Switch] $Force,
 
         [Switch] $IncludeSymbols
     )
 
     # Runtime and Configuration settings required by the package
-    ($Runtime, $Configuration) = if ($WindowsDownLevel) {
-        $WindowsDownLevel, "Release"
+    ($Runtime, $Configuration) = if ($WindowsRuntime) {
+        $WindowsRuntime, "Release"
```

Comment:
Maybe we don't need the `WindowsRuntime` at all. If `$IsWindows` is true, then it has to be `win7-x86` or `win7-x64`. About the architecture part, we can get that from the actual RID. See the example code below:
```
($Runtime, $Configuration) = New-PSOptions -Configuration "Release" -WarningAction SilentlyContinue | ForEach-Object { $_.Runtime, $_.Configuration }
if ($Environment.IsWindows) {
    $Runtime = $Runtime -replace "win\d+", "win7"
}
```